pipeline_dp/pipeline_operations.py

```diff
@@ -156,8 +157,14 @@ def map_tuple(self, col, fn, stage_name: str = None):
     def map_values(self, col, fn, stage_name: str):
         pass
 
-    def group_by_key(self, col, stage_name: str):
-        pass
+    def group_by_key(self, col, stage_name: str = None):
```

Comment:
change this to `stage_name: Optional[str]` to confor to mypy standards.

Reply:
Added it